test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-plugin.yaml.in

```diff
@@ -218,7 +218,7 @@ spec:
       serviceAccountName: csi-hostpathplugin-sa
       containers:
         - name: hostpath
-          image: k8s.gcr.io/sig-storage/hostpathplugin:v1.7.3
+          image: {{.HostpathpluginImage}}
```

Comment:
This substitution is based on the assumption that there will be only one hostpath plugin image. Same for the other images.

We may need the flexibility to reference different versions at some point. Right now that occurs for csi-resizer when considering both the hostpath deployment and gcp.